Makefile

```diff
@@ -292,15 +311,16 @@ clean-coverage:
 
 # Building Docker Images for our executables
 ############################################
-images: user-broker-image \
-    controller-manager-image apiserver-image
+images: user-broker-image controller-manager-image apiserver-image
 
 define build-and-tag # (service, image, mutable_image, prefix)
 	$(eval build_path := "$(4)build/$(1)")
 	$(eval tmp_build_path := "$(build_path)/tmp")
 	mkdir -p $(tmp_build_path)
 	cp $(BINDIR)/$(1) $(tmp_build_path)
-	docker build -t $(2) $(build_path)
+	cp $(build_path)/Dockerfile $(tmp_build_path)
+	sed -e "s|BASEIMAGE|$(BASEIMAGE)|g" -i $(tmp_build_path)/Dockerfile
```

Comment:
the `-i` will be the problem child. I think its supported on a mac but under a different flag - so try sed w/o modifying the file directly.